pandas/io/stata.py

```diff
@@ -1521,42 +1516,64 @@ def _read_value_labels(self) -> None:
             self._path_or_buf.seek(self._data_location + offset)
 
         self._value_labels_read = True
-        self._value_label_dict = {}
+        self._value_label_dict: dict[str, dict[int, str]] = {}
 
         while True:
             if self._format_version >= 117:
                 if self._path_or_buf.read(5) == b"</val":  # <lbl>
                     break  # end of value label table
 
-            slength = self._path_or_buf.read(4)
-            if not slength:
-                break  # end of value label table (format < 117)
-            if self._format_version <= 117:
-                labname = self._decode(self._path_or_buf.read(33))
+            if self._format_version >= 108:
+                slength = self._path_or_buf.read(4)
+                if not slength:
+                    break  # end of value label table (format < 117)
+                if self._format_version <= 108:
```

Comment:
This should be == 108 since you ahve >= 108 above and <= 108 means ==108.

Reply:
Yes, this should be equivalent (and clearer) - I was using the cut-off versions for label name lengths for reference. I'll make the suggested change.